diff --git a/include/target.mk b/include/target.mk
index 8341e4f..f2e653e 100644
--- a/include/target.mk
+++ b/include/target.mk
@@ -17,7 +17,7 @@
 block-mount coremark kmod-nf-nathelper kmod-nf-nathelper-extra kmod-ipt-raw wget libustream-openssl ca-certificates \
 default-settings luci luci-app-ddns luci-app-upnp luci-app-adbyby-plus luci-app-autoreboot \
 luci-app-filetransfer luci-app-vsftpd luci-app-ssr-plus luci-app-unblockmusic \
-luci-app-arpbind luci-app-vlmcsd luci-app-wol luci-app-ramfree \
+luci-app-arpbind luci-app-vlmcsd luci-app-wol \
 luci-app-sfe luci-app-flowoffload luci-app-nlbwmon luci-app-accesscontrol luci-app-cpufreq \
 ddns-scripts_aliyun ddns-scripts_dnspod
 # For nas targets
